ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.10.yaml

```diff
@@ -0,0 +1,334 @@
+base_images:
+  base:
+    name: "4.10"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.10"
+    namespace: ocp
+    tag: cli
+  openshift_knative-v1.0.0_knative-eventing-kafka-src:
+    name: knative-v1.0.0
+    namespace: openshift
+    tag: knative-eventing-kafka-src
+  openshift_knative-v1.0.0_knative-eventing-src:
+    name: knative-v1.0.0
+    namespace: openshift
+    tag: knative-eventing-src
+  openshift_knative-v1.0.1_knative-serving-src:
+    name: knative-v1.0.1
+    namespace: openshift
+    tag: knative-serving-src
+  openshift_release_golang-1.17:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+images:
+- context_dir: openshift/ci-operator/source-image
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    openshift_knative-v1.0.0_knative-eventing-kafka-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v1.0.0:knative-eventing-kafka-src
+    openshift_knative-v1.0.0_knative-eventing-src:
+      as:
+      - registry.ci.openshift.org/openshift/knative-v1.0.0:knative-eventing-src
```

Comment:
Same comment on the correct version

Reply:
@pierDipi This part (same as for your previous comment) is auto-generated. The references will be replaced very soon after this PR is merged. See the example commit here: https://github.com/openshift/release/commit/e21342d652d3f5f62238978ba0dbfb810adf058d
If you don't have other comments let's merge this and let the machinery do its job.